doc/nrf/doc_styleguide.rst

```diff
@@ -15,33 +15,91 @@ RST style guide
+
+Do not use consecutive headings without intervening text.
+
+Consecutive line rule
+=====================
+
+For readability reasons, start every sentence on a new line in the source files and do not add line breaks within the sentence.
+In the output, consecutive lines without blank lines in between are combined into one paragraph.
 
     .. note:: For the conceptual documentation written in RST, you can have more than 80 characters per line.
               The requirement for 80 characters per line applies only to the code documentation written in doxygen.
 
-Hyperlinks
-==========
-
-All external links must be defined in the ``links.txt`` file.
-Do not define them directly in the RST file.
-The reason for this is to allow for validating all links in one central location and to make it easy to update breaking links.
+Content highlighting
+====================
 
-Each link should be defined only once in ``links.txt``.
+Use the following highlighing rules in the RST conceptual documentation:
 
-If the link text that is defined in ``links.txt`` does not fit in the context where you use the link, you can override it by using the following syntax::
+.. content_highlighting_table:
 
-   `new link text <original link text_>`_
++------------------------+------------------------+---------------------------------------------------------+
+| Markup                 | Example                | Usage criteria                                          |
++========================+========================+=========================================================+
+| ``*emphasis*``         | *relaying*             | Emphasis or new terms.                                  |
++------------------------+------------------------+---------------------------------------------------------+
+| ````code````           | ``west update``        | Code within text, including file paths and names.       |
++------------------------+------------------------+---------------------------------------------------------+
+| ``**PCB**``            | **SW3**                | PCB names.                                              |
++------------------------+------------------------+---------------------------------------------------------+
+| ``:guilabel:`button``` | :guilabel:`Cancel`     | Clickable UI buttons.                                   |
++------------------------+------------------------+---------------------------------------------------------+
 
-It is also possible to define more than one default link text for a link, which can be useful if you frequently need a different link text::
+Do not use the following markup: ```..```.
 
-   .. _`Link text one`:
-   .. _`Link text two`: http://..
+For readability reasons, do not use any highlighting for titles of other pages or headings when they are mentioned in the text.
+For example, do not use emphasis to make the names of libraries stand out.
+Consider linking to these pages instead.
```

Comment:
We should ALWAYS link if we can, not only consider it.

Reply:
What about multiple links to the same library? Should we link only on first occurrence?

Reply:
First occurrence and any explicit pointer to it - so if we're talking about an external library, for example, we should have another link when we point out that the library must be enabled.